imagedecoder/pom.xml

```diff
@@ -6,8 +6,9 @@
     <artifactId>imagedecoder</artifactId>
     <version>0.9.0</version>
 
-    <name>Image Decoder Project (from https://github.com/mosip/imagedecoder)</name>
-
+    <name>imagedecoder</name>
```

Comment:
<name>Imagedecoder</name>
<url>http://github.com/mosip/imagedecoder</url>
<description>Image decoder for MOSIP</description>